src/mindlessgen/molecules/generate_molecule.py

```diff
@@ -518,7 +469,7 @@ def fixed_charge_elem_correction(
             if verbosity > 1:
                 print(f"Adding atom type {random_elem} for charge...")
             return natoms
-        elif natoms[random_elem] > min_count and num_atoms > cfg.min_num_atoms:
+        if natoms[random_elem] > min_count and num_atoms > cfg.min_num_atoms:
```

Comment:
Note for the review: The `elif` was considered unnecessary as a `return` statement was inevitable after the previous `if`